crates/pallets/starknet/src/lib.rs

```diff
@@ -87,6 +86,7 @@ pub mod pallet {
     use crate::types::{ContractStorageKeyWrapper, EthLogs, NonceWrapper, StarkFeltWrapper};
 
     #[pallet::pallet]
+	#[pallet::without_storage_info] // TODO: Remove this when we have bounded every storage item.
```

Comment:
Please create an issue for this todo

Reply:
https://github.com/keep-starknet-strange/madara/issues/93